src/v/cluster/service.cc

```diff
@@ -777,4 +777,23 @@ service::do_get_partition_state(partition_state_request req) {
       });
 }
 
+ss::future<describe_acls_reply>
+service::describe_acls(describe_acls_request&& req, rpc::streaming_context&) {
+    return ss::with_scheduling_group(
+             get_scheduling_group(),
+             [this, req]() mutable {
+                 return _topics_frontend.local().describe_acls(
+                   std::move(req.filter), req.timeout);
+             })
+      .then([](result<std::vector<acl_resource>> resources) mutable {
+          if (resources.has_error()) {
+              return describe_acls_reply{
+                .error_code = errc(resources.error().value())};
```

Comment:
This conversion feels a bit dirty. Is it guaranteed that `resources.error().category()` is always `cluster::errc_category`? I always feel like a `std::errc` of some sort could pop out of the network call.

Reply:
Or what if we make the error code in the reply a `std::errc` type?

Reply:
> Or what if we make the error code in the reply a `std::errc` type?

It ends up going through a `std::error_code` elsewhere.